catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -145,7 +145,6 @@
         <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-offline-catalog/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-feature/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf.spatial/spatial-commands/${project.version}</bundle>
-        <bundle>mvn:org.codice.ddf.spatial/spatial-geocoding-admin-module/${project.version}</bundle>
```

Comment:
Should _catalog/spatial/geocoding/spatial-geocoding-admin-module_ and _ui/packages/spatial-geocoding-admin-module_ be deleted too?

What is the motivation for removing the Admin Console part instead of fixing it?

Reply:
One of the endpoints used by the UI no longer exists. It was removed a long time ago and apparently no one noticed, so @clockard said to just remove the UI part.

Reply:
We should remove the modules/code as well. We don't want to leave dead code hanging around.

Reply:
Should catalog/spatial/geocoding/spatial-geocoding-admin-module be deleted too?

Reply:
@emmberk You're right! I thought that was the config, but that is elsewhere.

Reply:
✏️ Does this line need to be removed too? https://github.com/codice/ddf/blob/851f92fccfad58b54f960aa8b1b0c1bd773d1350/catalog/spatial/geocoding/pom.xml#L36